sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -614,6 +614,11 @@ export class MessageSender extends LinkEntity {
    * @returns {Promise<void>}
    */
   async send(data: SendableMessageInfo): Promise<void> {
+    if (!isJSONLikeObject(data) || data.body == undefined) {
+      throw new TypeError(
+        `Input must be a JS object and "body" property must not be undefined or null.`
+      );
```

Comment:
This would be a breaking change for existing users. I suggest not making this change.